### ISSUE TITLE
2.13-2.2.0 jar on Maven central seems broken

### ISSUE THREAD

**Issue by marko-asplund** [Kanaka-io/play-monadic-actions#46]
```
foo ~ % rm -f play-monadic-actions_2.13-2.2.0.jar
foo ~ % curl -sLO 'https://search.maven.org/remotecontent?filepath=io/kanaka/play-monadic-actions_2.13/2.2.0/play-monadic-actions_2.13-2.2.0.jar'
foo ~ % jar tf play-monadic-actions_2.13-2.2.0.jar
META-INF/MANIFEST.MF
foo ~ %
foo ~ % ls -la play-monadic-actions_2.13-2.2.0.jar
-rw-r--r--  1 foo  staff  308 Sep  3 18:31 play-monadic-actions_2.13-2.2.0.jar
```

**Comment by vil1** [comment]
I don't really get it, is the jar empty?

**Comment by marko-asplund** [comment]
Not entirely, but it only includes `MANIFEST.MF`.
You can verify this by downloading the jar using the curl command above.
AFAIK, the problem occurs during packaging and not an issue on the artifact repository.

**Comment by vil1** [comment]
That's pretty weird, I'll take a closer look (and probably redo the release, which will probably be a lot of fun :) )

**Comment by marko-asplund** [comment]
It looks like the packaging in version 2.2.0 is broken for Scala 2.11 and 2.12:
```

foo@bar play-monadic-actions % sbt '+publishLocal' | ggrep -oP 'published .* to \K.*/jars/play-monadic-actions_.*.jar' | xargs ls -la
-rw-r--r--  1 foo  staff    319 Sep 18 23:22 /home/foo/.ivy2/local/io.kanaka/play-monadic-actions_2.11/2.2.0-SNAPSHOT/jars/play-monadic-actions_2.11.jar
-rw-r--r--  1 foo  staff    319 Sep 18 23:22 /home/foo/.ivy2/local/io.kanaka/play-monadic-actions_2.12/2.2.0-SNAPSHOT/jars/play-monadic-actions_2.12.jar
-rw-r--r--  1 foo  staff  33286 Sep 18 23:22 /home/foo/.ivy2/local/io.kanaka/play-monadic-actions_2.13/2.2.0-SNAPSHOT/jars/play-monadic-actions_2.13.jar
-rw-r--r--  1 foo  staff  33286 Sep 18 23:22 /home/foo/.ivy2/local/io.kanaka/play-monadic-actions_2.13/2.2.0-SNAPSHOT/jars/play-monadic-actions_2.13.jar
-rw-r--r--  1 foo  staff  33286 Sep 18 23:22 /home/foo/.ivy2/local/io.kanaka/play-monadic-actions_2.13/2.2.0-SNAPSHOT/jars/play-monadic-actions_2.13.jar
-rw-r--r--  1 foo  staff  33286 Sep 18 23:22 /home/foo/.ivy2/local/io.kanaka/play-monadic-actions_2.13/2.2.0-SNAPSHOT/jars/play-monadic-actions_2.13.jar
```

**Comment by marko-asplund** [comment]
and for version 2.1.0
```
foo@bar play-monadic-actions % sbt '+publishLocal' | ggrep -oP 'published .* to \K.*/jars/play-monadic-actions_.*.jar' | xargs ls -la
-rw-r--r--  1 foo  staff  46773 Sep 18 23:40 /home/foo/.ivy2/local/io.kanaka/play-monadic-actions_2.11/2.1.0/jars/play-monadic-actions_2.11.jar
-rw-r--r--  1 foo  staff  46773 Sep 18 23:40 /home/foo/.ivy2/local/io.kanaka/play-monadic-actions_2.11/2.1.0/jars/play-monadic-actions_2.11.jar
-rw-r--r--  1 foo  staff  31148 Sep 18 23:40 /home/foo/.ivy2/local/io.kanaka/play-monadic-actions_2.12/2.1.0/jars/play-monadic-actions_2.12.jar
-rw-r--r--  1 foo  staff  31148 Sep 18 23:40 /home/foo/.ivy2/local/io.kanaka/play-monadic-actions_2.12/2.1.0/jars/play-monadic-actions_2.12.jar
```
So, the issue seems to have been introduced with version 2.2.0 and can be reproduced locally without publishing to Maven central or other external artifact repository.

**Comment by marko-asplund** [comment]
This issue is addressed in PR Kanaka-io/play-monadic-actions#47.
Though it drops Scala 2.11 support I'd argue that it improves the current situation since currently packaging only works for Scala 2.13 and after the change for versions 2.12 and 2.13.

**Comment by marko-asplund** [comment]
Looking at play-monadic-actions v2.2.0 packages on Maven central, it looks like packages for Scala v2.11 and v2.12 didn't get published at all and the Scala 2.13 package was broken for some reason.

To build against all versions listed in `crossScalaVersions`, the `+` prefix is required.
Was this prefix used when publishing? If it was omitted, it together with an [issue](https://github.com/Kanaka-io/play-monadic-actions/blob/3d28be479e6916267985dea35fe8069f5ca9ca45/build.sbt#L19) in `build.sbt` could explain why Scala 2.11 and 2.12 packages weren't built.

As for the issue with the Scala 2.13 package, maybe it was a transient error on Maven central.
Could you please try publishing again?

**Comment by marko-asplund** [comment]
Any news on this?

**Comment by vil1** [comment]
As of right now, version `2.2.1` has been successfully (and correctly) published for Scala 2.11, 2.12 and 2.13. Moreover, upcoming releases will be automatic 🎊

**Comment by marko-asplund** [comment]
Just tried and the Scala 2.12 and 2.13 jars on Maven central look to be working now.
Thanks! 👍 😄 🎉
